script.py

```diff
@@ -204,6 +210,28 @@ def _add_tracks_to_tidal_playlist(playlist_id, tracks):
                 print('Error adding tracks to playlist: ' + e)
                 # TODO: should add playlist name to CSV of failures
 
+def get_tidal_old_playlists():
```

Comment:
Haha, this is great. I didn't write something like this because I was lazy, so I had a million versions of the same playlist in Tidal by the time I finished testing all my code.

Reply:
It would be better to compare the tracks that are in the two playlists (Spotify & TIDAL) and remove/add tracks when needed. If I'll have time in the future, I will try to implement this behaviour.

Reply:
Yeah, it would be more efficient. I didn't bother with doing anything other than a dumb copy because I was mainly looking to copy everything from Spotify to Tidal over just once, and then never using Spotify again. In retrospect, I would like to add in the ability to keep Spotify synced with Tidal (adding songs to Spotify playlists based on changes to the equivalent Tidal playlist, adding likes to Spotify based on new likes from Tidal), which would benefit from a more efficient method to copy songs over.